spoken/api.py

```diff
@@ -22,7 +22,8 @@ def get(self, request, format=None):
             context["language"] = search_language
             context["foss_languages"] = get_foss_languages(search_foss)
             context["tutorials"] = get_tutorials_completion_status(search_foss, search_language, request.user) if request.user.is_authenticated else get_tutorials(search_foss, search_language)
-            
+        if request.user.is_authenticated:
+            context['courseProgress']=get_course_progress(request)
```

Comment:
This logic will not work when a unauthenticated user will search for tutorials. Please resolve this

Reply:
There is already a line of code if request.user.is_authenticated: in class before. One similar logic is making it redundant.